CMakeLists.txt

```diff
@@ -680,6 +684,7 @@ if(LIBRE_BUILD_SHARED)
     ${PROJECT_SOVERSION}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH})
   set_target_properties(re-shared PROPERTIES SOVERSION ${PROJECT_SOVERSION})
   set_target_properties(re-shared PROPERTIES OUTPUT_NAME "re")
+  add_library(libre::re-shared ALIAS re-shared)
```

Comment:
should the application know if the library is shared or static ?

for example OpenSSL alias is "OpenSSL::SSL" ...